src/istgt_lu.h

```diff
@@ -326,6 +328,7 @@ typedef struct istgt_lu_t {
 	ISTGT_LU_MAP map[MAX_LU_MAP];
 	int conns;
 #ifdef REPLICATION
+	uint64_t replica_id[MAXREPLICA];
```

Comment:
why is this required?

Reply:
ok. My base branch was different. That's why this change is here. I will remove this change.